test/evm/hardhat/chain-specific-spokepools/ZkSync_SpokePool.ts

```diff
@@ -109,6 +124,33 @@ describe("ZkSync Spoke Pool", function () {
     expect(zkErc20Bridge.withdraw).to.have.been.calledOnce;
     expect(zkErc20Bridge.withdraw).to.have.been.calledWith(hubPool.address, l2Dai, amountToReturn);
   });
+  it("Bridge tokens to hub pool correctly calls the Standard L2 Bridge for zkSync Bridged USDC.e", async function () {
+    // Redeploy the SpokePool with usdc address -> 0x0
+    const usdcAddress = ZERO_ADDRESS;
+    const constructorArgs = [weth.address, usdcAddress, zkUSDCBridge.address, cctpTokenMessenger, 60 * 60, 9 * 60 * 60];
+    const implementation = await hre.upgrades.deployImplementation(
+      await getContractFactory("ZkSync_SpokePool", owner),
+      { kind: "uups", unsafeAllow: ["delegatecall"], constructorArgs }
+    );
+    await zkSyncSpokePool.connect(crossDomainAlias).upgradeTo(implementation);
+
+    const { leaves, tree } = await constructSingleRelayerRefundTree(l2Usdc, await zkSyncSpokePool.callStatic.chainId());
+    await zkSyncSpokePool.connect(crossDomainAlias).relayRootBundle(tree.getHexRoot(), mockTreeRoot);
+    await zkSyncSpokePool.connect(relayer).executeRelayerRefundLeaf(0, leaves[0], tree.getHexProof(leaves[0]));
+
+    // This should have sent tokens back to L1. Check the correct methods on the gateway are correctly called.
+    expect(zkErc20Bridge.withdraw).to.have.been.calledOnce;
+    expect(zkErc20Bridge.withdraw).to.have.been.calledWith(hubPool.address, l2Usdc, amountToReturn);
+  });
+  it("Bridge tokens to hub pool correctly calls the custom USDC L2 Bridge for Circle Bridged USDC", async function () {
+    const { leaves, tree } = await constructSingleRelayerRefundTree(l2Usdc, await zkSyncSpokePool.callStatic.chainId());
+    await zkSyncSpokePool.connect(crossDomainAlias).relayRootBundle(tree.getHexRoot(), mockTreeRoot);
+    await zkSyncSpokePool.connect(relayer).executeRelayerRefundLeaf(0, leaves[0], tree.getHexProof(leaves[0]));
+
+    // This should have sent tokens back to L1. Check the correct methods on the gateway are correctly called.
+    expect(zkUSDCBridge.withdraw).to.have.been.calledOnce;
```

Comment:
I'm not sure how this test passes given the mistake [here](https://github.com/across-protocol/contracts/pull/941/files#r2020256782)